src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -2080,23 +2100,37 @@ type Place {
     """The stop related to the place."""
     stop: Stop
 
-    """The bike rental station related to the place"""
-    bikeRentalStation: BikeRentalStation @deprecated(reason: "Use vehicleRentalStation and rentalVehicle instead")
+    """
+    The position the stop in the pattern. This is not required to start from 0 or be consecutive - any
```

Comment:
```suggestion
    The position of the stop in the pattern. This is not required to start from 0 or be consecutive - any
```

Reply:
Should we update this field documentation so it's more generic for the two possible return types.